task2.py

```diff
@@ -0,0 +1,38 @@
+"""
+Include tasks for basic data types and  exception handling
+"""
+
+import sys
+
+
+def multiply_digits_of_number(number):
+    """
+    Returns product of digits in a number
+    """
+    if not isinstance(number, int):
+        raise TypeError('Must be an integer')
+
+    product = 1
+    while number:
+        digit = number % 10
+        product *= digit
+        number //= 10
+
+    return product
+
+
+def main(argv):
+    """
+    Main entry point of program
+    """
+    try:
+        argument = int(argv[1])
```

Comment:
You may take a look at `argparse` https://docs.python.org/3/howto/argparse.html
it is a standard for parsing command line arguments